google-cloud-storage/src/main/java/com/google/cloud/storage/JsonResumableSessionFailureScenario.java

```diff
@@ -79,6 +78,7 @@ enum JsonResumableSessionFailureScenario {
           .or(matches("Content-Type"))
           .or(matches("Range"))
           .or(startsWith("X-Goog-Stored-"))
+          .or(matches("X-Goog-GCS-Idempotency-Token"))
```

Comment:
What if the code collapsed to `startsWith("X-Goog-")`?

Reply:
I wanted to reduce the scope to prevent an accidental inclusion of a header that might contain a sensitive value.

If it were relaxed to `x-goog-` that could pull in object metadata fields which could have sensitive values.

Since these error messages go into peoples logs, we should only include what we know is safe.